api/appendix_e.asciidoc

```diff
@@ -457,11 +457,11 @@ conformance process:
 
   * {CL_DEVICE_LATEST_CONFORMANCE_VERSION_PASSED}
 
-== Summary of Changes from OpenCL 3.0
+== Summary of Changes to OpenCL 3.0
```

Comment:
If you're looking for suggestions, could also say "Changes **after** OpenCL 3.0", but I'm not sure if that's any better.